master/buildbot/test/e2e/test_master_worker_create_start_stop.py

```diff
@@ -0,0 +1,921 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+from __future__ import print_function
+
+import os
+import re
+import socket
+import textwrap
+
+from fastjsonrpc.client import Proxy
+from fastjsonrpc.client import jsonrpc
+from twisted.internet import reactor as global_reactor
+from twisted.internet import defer
+from twisted.internet import task
+from twisted.internet import utils
+from twisted.python import log
+from twisted.trial import unittest
+from txrequests import Session
+
+from buildbot.test.util import dirs
+from buildbot.test.util.decorators import skipUnlessInstalled
+from buildbot.test.util.decorators import skipUnlessPlatformIs
+
+try:
+    from shutil import which
```

Comment:
this is a standard library module, so the whole thing should go *before* 3rd party modules.